### DISCUSSION TITLE
How to do Design Customisation after install the plugnmeet in AWS EC2 instance.

### DISCUSSION THREAD

**Discussion by ThilinaV98** [mynaparrot/plugNmeet-server#341] [Q&A] [answered]
I already installed the pulgNmeet in AWS EC2 instance. now I want to do some design customisation and I tried to change the 
window.DESIGN_CUSTOMIZATION = `{
  "primary_color": "#004D90",
  "secondary_color": "#24AEF7",
  "background_color": "#0b7db4",
  "background_image": "https:\/\/mydomain.com\/custom_bg.png",
  "header_bg_color": "#45b3ec",
  "footer_bg_color": "#45b3ec",
  "left_side_bg_color": "#04a2f3",
  "right_side_bg_color": "#04a2f3",
  "custom_css_url": "https:\/\/mydomain.com\/plugNmeet_desing.css"
}`; 
values in config.js flie from /opt/plugNmeet/client/dist/assets and restart the plugnmeet service but it didn't work.  

how can I do the Design Customisation ?

**Comment by jibon57** [comment, marked as answer]
Changing in `config.js` will only work if you use plugNmeet server's static file directly e.g https://your_plugNmeet.com/?access_token=token
but if you've your own page then require to add those value manually like other fields e.g [conference.php](https://github.com/mynaparrot/plugNmeet-sdk-php/blob/main/examples/conference.php). Also make sure you've cleared your browser cache to get the new update of `config.js` file.

**Comment by ThilinaV98** [comment]
Yes, I used plugNmeet server's static file directly like https://my_plugNmeet.com/?access_token=token and change the values in config.js file to design customisation and cleared browser cache but still getting previous colors.

**Reply by jibon57** [reply]
Verify that the JSON code is valid from here: https://jsonlint.com/

**Reply by jibon57** [reply]
Also you'll need to use `backtick` because it's not an object. https://github.com/mynaparrot/plugNmeet-client/blob/0e51e32d4a5584886b44cae721117ec15ad7826d/src/assets/config_sample.js#L36C4-L37C1
